.github/workflows/coredump.yml

```diff
@@ -40,16 +40,20 @@ jobs:
       - name: Run Ruby script
         id: ruby-script
         run: |
-          ruby ci/ruby-programs/infinite_on_cpu.rb &
+          ruby ci/ruby-programs/cme_complex_labels.rb &
           pid=$!
           echo "pid=$pid" >> $GITHUB_OUTPUT
 
       - name: Inspect process
         run: sudo ./target/release/rbspy inspect --pid ${{ steps.ruby-script.outputs.pid }}
 
+      # default coredump filter is 0x33, update it to basically dump all memory here
+      # to best mimic the real process otherwise heap data like strings may not
+      # be accessible
       - name: Create core dump
         shell: sudo bash {0}
         run: |
+          echo 0x7f > /proc/${{ steps.ruby-script.outputs.pid }}/coredump_filter
```

Comment:
Note that 0xff also seems to work, and it might also be a good idea to do gcore -a 

It drove me crazy trying to figure out why my string reads would work on a local process, but fail in the coredump - the memory region the string was in was not in the coredump lol.

Github uses 0x33 by default for coredump filter.